sklearn/externals/joblib/pool.py

```diff
@@ -68,7 +68,7 @@ class CustomizablePickler(Pickler):
     # We override the pure Python pickler as its the only way to be able to
     # customize the dispatch table without side effects in Python 2.7
     # to 3.2. For Python 3.3+ leverage the new dispatch_table
-    # feature from https://bugs.python.org/issue14166 that makes it possible
+    # feature from http://bugs.python.org/issue14166 that makes it possible
```

Comment:
all the https->http changes are a bit confusing?

Reply:
ohhh did we change them with a regex in scikit-learn?

Reply:
The HTTPS switch was not contributed upstream to joblib, only in scikit-learn.

Reply:
Fixed upstream: https://github.com/joblib/joblib/pull/805 (for the next release).